virtio-devices/src/transport/pci_device.rs

```diff
@@ -800,7 +800,11 @@ impl VirtioPciDevice {
 
             queues.push((
                 queue_index,
-                vm_virtio::clone_queue(queue),
+                vm_virtio::clone_queue(
```

Comment:
It would be better to make sure that the addresses have already run through the `AccessPlatform` before they are turned into a `Queue`.

Reply:
I think those already gets translated at the per device level using the SevSnpProxy.